scripts/osbuilder.sh

```diff
@@ -116,7 +116,7 @@ build_rootfs()
 	fi
 	DNF="$DNF --config=$DNF_CONF -y --installroot=${ROOTFS_DIR} --noplugins"
 	$DNF install systemd hyperstart cc-oci-runtime-extras coreutils-bin \
-		systemd-bootchart iptables-bin clear-containers-agent ${EXTRA_PKGS}
+		systemd-bootchart iptables-bin clear-containers-agent bash-bin ${EXTRA_PKGS}
```

Comment:
does the agent need bash ?

Reply:
hey @devimc, it does not. It a request to allow to debug the vm.

Reply:
my vote that if this is only for debug the developer build their own image (with this tool) with bash included, and we do not need to ship with bash for security risks.

Documentation will fix that IMHO

Reply:
@devimc @gorozco1 @jodh-intel 

How is the security risk different between us including as shell and the user of the container who normally already has access to a shell as part of the container workload. If there is a material difference between the two then it makes a difference. If not we are just sacrificing usability.

So the question is, how is this shell different from the shell available in the workload. Both are root. I

Reply:
@mcastelino well, if we provide bash this is the `vm` rootfs, the shell that is provided with the workload is in  another layer, limited within the container - imho - correct me if I am wrong.